### ISSUE TITLE
Potential issues getting started following README

### ISSUE THREAD

**Issue by stevegrossi** [supernullset/hedwig_flowdock#7]
I'm on Elixir 1.3.2 with Erlang 19, following the README here, and having some trouble. I'd be happy to submit a PR to fix these if they are in fact issues.
### Required version incorrectly specified

First, when I specify the dependency following the README's example:

``` elixir
def deps do
  [{:hedwig_flowdock, "0.1"}]
end
```

I get the error from Mix:

```
$ mix deps.get
** (Mix) Required version "0.1" for package hedwig_flowdock is incorrectly specified (from: mix.exs)
```

I suspect it may need to use the pessimistic constraint (`~> 0.1`) or specify the exact version on [Hex.pm](https://hex.pm/packages/hedwig_flowdock):

``` elixir
def deps do
  [{:hedwig_flowdock, "0.1.2"}]
end
```

That appears to work.
### Hedwig.Adapter is not loaded and could not be found

At this point, the README says to run `mix hedwig.gen.robot`, however for me that throws a compilation error:

```
== Compilation error on file lib/hedwig_flowdock.ex ==
** (CompileError) lib/hedwig_flowdock.ex:2: module Hedwig.Adapter is not loaded and could not be found
    (elixir) expanding macro: Kernel.use/1
    lib/hedwig_flowdock.ex:2: Hedwig.Adapters.Flowdock (module)
    (elixir) lib/kernel/parallel_compiler.ex:116: anonymous fn/4 in Kernel.ParallelCompiler.spawn_compilers/1

could not compile dependency :hedwig_flowdock, "mix compile" failed. You can recompile this dependency with "mix deps.compile hedwig_flowdock", update it with "mix deps.update hedwig_flowdock" or clean it with "mix deps.clean hedwig_flowdock"
```

`module Hedwig.Adapter is not loaded` suggested to me that perhaps the base [hedwig](https://github.com/hedwig-im/hedwig) library is also required. I expected it to be a dependency of `hedwig_flowdock`, but looking in `mix.lock` it doesn't appear to be present. If it's not a dependency, perhaps the `deps` block in the README should indicate that it's meant to be listed separately?

``` ruby
defp deps do
  [
    {:hedwig, "0.3.0"},
    {:hedwig_flowdock, "0.1.2"}
  ]
end
```

Of course, I could be wrong, because I still get the same compilation error when running `mix hedwig.gen.robot`:

```
module Hedwig.Adapter is not loaded and could not be found
```

Have I made some mistake, or might these be real issues?

**Comment by janxious** [comment]
``` elixir
defmodule HedwigTest.Mixfile do
  use Mix.Project

  def project do
    [app: :hedwig_test,
     version: "0.1.0",
     elixir: "~> 1.3",
     build_embedded: Mix.env == :prod,
     start_permanent: Mix.env == :prod,
     deps: deps()]
  end

  # Configuration for the OTP application
  #
  # Type "mix help compile.app" for more information
  def application do
    [applications: [:logger, :hedwig, :hedwig_flowdock]]
  end

  # Dependencies can be Hex packages:
  #
  #   {:mydep, "~> 0.3.0"}
  #
  # Or git/path repositories:
  #
  #   {:mydep, git: "https://github.com/elixir-lang/mydep.git", tag: "0.1.0"}
  #
  # Type "mix help deps" for more examples and options
  defp deps do
    [{:hedwig, github: "hedwig-im/hedwig"},
     {:hedwig_flowdock, "~> 0.1"}]
  end
end
```

Doing that, I got a working mix project that can run the generator. I do agree the instructions could be more clear.

**Comment by supernullset** [comment]
@stevegrossi @janxious I am SO SO sorry that the notifications on this thread got lost in my inbox. I will get the README updated to be more clear as I am in here and doing some cleanup work.

**Comment by stevegrossi** [comment]
No problem! And thanks so much for your hard work on `hedwig_flowdock` 👍

**Comment by supernullset** [comment]
Closed by #9
